### DISCUSSION TITLE
Добавить опцию возвращения системы иммунитетов из ТЧ

### DISCUSSION THREAD

**Discussion by MsMarshie** [OGSR/OGSR-Engine#363] [Ideas]
В моём моде есть артефакт, который позволяет лечиться в кислоте за счёт 150% иммунитета к хим.ожогу. Когда решил переносить на OGSR Engine, заметил, что иммунитеты теперь кэпаются на 100%. Можно ли будет добавить опцию, чтобы иммунитеты снова работали, как в ТЧ - то есть не имели кэпа и лечили, если достигают выше 100%? Спасибо.

**Comment by I-AmDead** [comment]
> В моём моде есть артефакт, который позволяет лечиться в кислоте за счёт 150% иммунитета к хим.ожогу. Когда решил переносить на OGSR Engine, заметил, что иммунитеты теперь кэпаются на 100%. Можно ли будет добавить опцию, чтобы иммунитеты снова работали, как в ТЧ - то есть не имели кэпа и лечили, если достигают выше 100%? Спасибо.

А то что это костыль тебя не парит?

**Reply by MsMarshie** [reply]
Нет конечно, какое васянство без костылей?
Ну если ж атакуете пассивно-агрессивностью - извольте пояснить, как сделать некостыльно.

**Reply by I-AmDead** [reply]
> Нет конечно, какое васянство без костылей? Ну если ж атакуете пассивно-агрессивностью - извольте пояснить, как сделать некостыльно.

Сделать к примеру проверку что данный артефакт на поясе и гг получает такой то такой то урон, далее уже делаешь свои манипуляции. Это самое первое что в голову сбрело ( возможно даже совсем не грамотно, но уж лучше чем у вас). И никто вас не аттакует агрессивно, я лишь хотел довести до вас что это баг и лепить на нём какие то механики не стоит.

**Reply by MsMarshie** [reply]
Вообще, учитывая достаточно общепринятую страту с четыремя Батарейками в ТЧ - я бы сказал, что это скорее багофича. Но спасибо за предложения, посмотрю, что можно сделать.
